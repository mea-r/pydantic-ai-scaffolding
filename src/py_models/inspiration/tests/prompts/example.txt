Generate a concise and powerful inspirational quote about the virtue of perseverance.