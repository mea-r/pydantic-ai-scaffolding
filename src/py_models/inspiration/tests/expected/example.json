{
  "quote": "The difference between a successful person and others is not a lack of strength, not a lack of knowledge, but rather a lack in will.",
  "author": "Vince Lombardi"
}